Components/Rating/rating.css

```diff
@@ -2,5 +2,5 @@
 
 }
 .rated{
-    color: #f8ea22;
+    color: #ffbb28;
```

Comment:
try to use color variable

Reply:
Thanks for pointing it out. I will change those.